frontend/src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/RestrictiveLegalValues/RestrictiveLegalValues.tsx

```diff
@@ -33,13 +36,14 @@ const createValuesMap = (values: string[]): IValuesMap => {
 };
 
 export const RestrictiveLegalValues = ({
```

Comment:
when we're fixing a bug I'd add a test showing that we actually fixed a bug. This component seems to be quite easy to test.

Reply:
added